src/sentry/integrations/slack/requests/options_load.py

```diff
@@ -18,6 +18,8 @@ class SlackOptionsLoadRequest(SlackRequest):
 
     @property
     def group_id(self) -> int:
+        if self.data["container"].get("is_app_unfurl"):
+            return int(json.loads(self.data["app_unfurl"]["blocks"][0]["block_id"])["issue"])
```

Comment:
for carrying out assignment on unfurl issues in slack, see note above about testing

Reply:
Is `container` guaranteed? You could do `self.data.get("container", {}).get("is_app_unfurl")` to be safe